### ISSUE TITLE
Fix deprecation warning

### ISSUE THREAD

**Issue by milost** [ibalazevic/HypER#6]
The `nn.functional` module is deprecated and should be replaced by using `torch` directly. 

During execution the following warning is thrown:
`UserWarning: nn.functional.sigmoid is deprecated. Use torch.sigmoid instead.`
